fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/metadata/expressions/FunctionKeyExpression.java

```diff
@@ -282,7 +279,8 @@ public int planHash(@Nonnull final PlanHashKind hashKind) {
                 return getName().hashCode() + getArguments().planHash(hashKind);
             case FOR_CONTINUATION:
             case STRUCTURAL_WITHOUT_LITERALS:
-                return PlanHashable.objectsPlanHash(hashKind, BASE_HASH, getName(), getArguments());
+                // No BASE_HASH for abstract classes
```

Comment:
Would it be better to remove this case and therefore throw an error when not overridden?